src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -114,6 +121,11 @@ public static void removeBiome(BiomeType type, BiomeEntry entry)
         }
     }
 
+    public static void removeNetherBiome(Biome biome)
```

Comment:
Lacking documentation, when should this be called? I assume you intended `FMLCommonSetupEvent`?